2022-07-19 12:19:18,373 ERROR: Exception on /maps [GET] [in /mnt/e/microblog/venv/lib/python3.8/site-packages/flask/app.py:1455]
Traceback (most recent call last):
  File "/mnt/e/microblog/venv/lib/python3.8/site-packages/flask/app.py", line 2077, in wsgi_app
    response = self.full_dispatch_request()
  File "/mnt/e/microblog/venv/lib/python3.8/site-packages/flask/app.py", line 1525, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/mnt/e/microblog/venv/lib/python3.8/site-packages/flask/app.py", line 1523, in full_dispatch_request
    rv = self.dispatch_request()
  File "/mnt/e/microblog/venv/lib/python3.8/site-packages/flask/app.py", line 1509, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "/mnt/e/microblog/app/routes.py", line 220, in maps
    return render_template ('index copy.html')
  File "/mnt/e/microblog/venv/lib/python3.8/site-packages/flask/templating.py", line 149, in render_template
    ctx.app.jinja_env.get_or_select_template(template_name_or_list),
  File "/mnt/e/microblog/venv/lib/python3.8/site-packages/jinja2/environment.py", line 1081, in get_or_select_template
    return self.get_template(template_name_or_list, parent, globals)
  File "/mnt/e/microblog/venv/lib/python3.8/site-packages/jinja2/environment.py", line 1010, in get_template
    return self._load_template(name, globals)
  File "/mnt/e/microblog/venv/lib/python3.8/site-packages/jinja2/environment.py", line 969, in _load_template
    template = self.loader.load(self, name, self.make_globals(globals))
  File "/mnt/e/microblog/venv/lib/python3.8/site-packages/jinja2/loaders.py", line 138, in load
    code = environment.compile(source, name, filename)
  File "/mnt/e/microblog/venv/lib/python3.8/site-packages/jinja2/environment.py", line 768, in compile
    self.handle_exception(source=source_hint)
  File "/mnt/e/microblog/venv/lib/python3.8/site-packages/jinja2/environment.py", line 936, in handle_exception
    raise rewrite_traceback_stack(source=source)
  File "/mnt/e/microblog/app/templates/index copy.html", line 48, in template
    <script src="{% static 'app/templates/index.js' %}"></script>
jinja2.exceptions.TemplateSyntaxError: Encountered unknown tag 'static'.
2022-07-19 12:28:34,411 INFO: Microblog startup [in /mnt/e/microblog/app/__init__.py:50]
2022-07-19 12:29:55,587 INFO: Microblog startup [in /mnt/e/microblog/app/__init__.py:50]
2022-07-19 12:30:59,513 INFO: Microblog startup [in /mnt/e/microblog/app/__init__.py:50]
2022-07-19 12:35:24,571 INFO: Microblog startup [in /mnt/e/microblog/app/__init__.py:50]
2022-07-19 12:38:32,408 INFO: Microblog startup [in /mnt/e/microblog/app/__init__.py:50]
2022-07-19 12:41:55,972 INFO: Microblog startup [in /mnt/e/microblog/app/__init__.py:50]
2022-07-19 12:45:47,715 INFO: Microblog startup [in /mnt/e/microblog/app/__init__.py:50]
2022-07-19 12:46:50,941 INFO: Microblog startup [in /mnt/e/microblog/app/__init__.py:50]
2022-07-19 12:56:40,624 INFO: Microblog startup [in /mnt/e/microblog/app/__init__.py:50]
